### DISCUSSION TITLE
Dependabot: Dismiss an Alert at Org Level

### DISCUSSION THREAD

**Discussion by AnthonyHerman** [community/community#17621] [Code Security]
Our org has started to review some of the noisier ones for our environment and the ability to dismiss a specific alert we accept risk on org wide (without iterating through thousands of repos) would be nice.

So, for all instances of "x" - allow for a way to dismiss these across all repos with such a dependabot alert.

**Comment by sduquette-devolutions** [comment]
Absolutely. Short of that, being able to dismiss multiple issues via the panel at https://github.com/orgs/<orgName>security/alerts/dependabot would be a big QoL improvement.
